packages/components/src/styles/global.css

```diff
@@ -1,8 +1,7 @@
-
```

Comment:
prettier changes -- think `stylelint-config-styled-components` was suppressing some of these rules